### PR TITLE
Avoid reprocessing certain accessions as specified in a YAML file

### PR DESCRIPTION
**Pull request by pmb59**

### CODE REVIEW

File: Snakefile-sorting-hat [hunk 1]
```diff
@@ -104,6 +104,28 @@ def get_db_params(param):
     else:
         sys.exit(1)
 
+def read_skip_accessions_file(analysis_goal):
+    """
+    List of accessions that should not skipped from reprocessing
```
Comment (anilthanki) [inline Snakefile-sorting-hat]: You mean List of accessions that **should be skipped** or **should not be skipped** from reprocessing.

File: Snakefile-sorting-hat [hunk 2]
```diff
@@ -104,6 +104,28 @@ def get_db_params(param):
...
+                except yaml.YAMLError as exc:
+                    print(exc)
+            return skip_accessions_reprocessing['skips']['accession']
+        else:
+            return None
+    elif analysis_goal == 'recalculations':
+        # not implemented, not necessary so far
+        return None
+    else:
+        return None
```
Comment (anilthanki) [inline Snakefile-sorting-hat:127]: May be add some informative message here and at line 124

File: Snakefile-sorting-hat (hunk 1 above)
Comment (pmb59) [reply Snakefile-sorting-hat]: should be skipped - thanks [5031409](https://github.com/ebi-gene-expression-group/bulk-recalculations/pull/55/commits/5031409f2b19c2da9aaee321b6cd598f48579370)

**Comment by pmb59** [comment]
thanks the review @anilthanki

**Review by anilthanki — APPROVED** [review]

**Comment by anilthanki** [comment]
LGTM...

**Pull request merged**